commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemLocalServiceImpl.java

```diff
@@ -557,6 +557,32 @@ public CommerceOrderItem updateCommerceOrderItemPrice(
 		return commerceOrderItemPersistence.update(commerceOrderItem);
 	}
 
+	@Indexable(type = IndexableType.REINDEX)
+	@Override
+	public CommerceOrderItem updateCommerceOrderItemPrices(
+			long commerceOrderItemId, BigDecimal unitPrice,
+			BigDecimal promoPrice, BigDecimal discountAmount,
+			BigDecimal finalPrice, BigDecimal discountPercentageLevel1,
+			BigDecimal discountPercentageLevel2,
+			BigDecimal discountPercentageLevel3,
+			BigDecimal discountPercentageLevel4)
+		throws PortalException {
+
+		CommerceOrderItem commerceOrderItem =
+			commerceOrderItemPersistence.findByPrimaryKey(commerceOrderItemId);
+
+		commerceOrderItem.setUnitPrice(unitPrice);
```

Comment:
Hey @alecsloan I think we should also be updating the `modifiedDate` here.

Reply:
That's not auto handled by persistence?

Reply:
Looks like it should be:
```
if (!commerceOrderModelImpl.hasSetModifiedDate()) {
	if (serviceContext == null) {
		commerceOrder.setModifiedDate(now);
	}
	else {
		commerceOrder.setModifiedDate(
			serviceContext.getModifiedDate(now));
	}
}

Reply:
Woah no way! That didn't use to be there :)